app/src/test/java/com/example/samplegallery/LruBitmapCacheTest.java

```diff
@@ -0,0 +1,89 @@
+package com.example.samplegallery;
+
+/**
+ * Created by Kadiyala on 1/16/17.
+ */
+
+import android.content.Context;
+import android.util.DisplayMetrics;
+
+import com.example.samplegallery.Utilities.*;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
I would put all your static imports first, then organize the non static imports alphabetically. Usually only space between static and non static imports, but I'm not sure on this?